includes/SmallElementsHooks.php

```diff
@@ -37,6 +37,9 @@ public function onGetPreferences( $user, &$preferences ) {
 	 * @inheritDoc
 	 */
 	public function onOutputPageBodyAttributes( $out, $sk, &$bodyAttrs ): void {
+		if ( $sk->getSkinName() !== 'femiwiki' ) {
```

Comment:
This skin defines and uses `\MediaWiki\Skins\Femiwiki\Constants::SKIN_NAME`, so
```suggestion
		if ( $sk->getSkinName() !== Constants::SKIN_NAME ) {
```
will be good.